Symfony/CS/Fixer/Contrib/PhpdocVarToTypeFixer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+class PhpdocVarToTypeFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $token) {
+            $doc = new DocBlock($token->getContent());
+
+            foreach ($doc->getAnnotationsOfType('var') as $annotation) {
+                $line = $doc->getLine($annotation->getStart());
+                $line->setContent(str_replace('@var', '@type', $line->getContent()));
+            }
+
+            $token->setContent($doc->getContent());
```

Comment:
You can early skip iteration if there are no `var` annotation.
Then there will be no nee for regenerating content ;)